docs/GettingStarted.md

```diff
@@ -583,6 +612,34 @@ On the server:
       end
     end
 
+### Troubleshooting
+
+#### Logs
+
+Your application log should contain informations and report problems
+such as agent not being up and running. All logs generated by this
+library should contain ``[ddtrace]``. Also see [how to use a custom logger](#Using_a_custom_logger)
+to redirect these messages elsewhere.
+
+The [Datadog Trace Agent](https://github.com/DataDog/datadog-trace-agent)
+should by default be listening for traces on port 8126.
+
+#### Hello World
+
+Sometimes, setting up a complete application is complex, so in doubt,
+try the small program below, which should be able to report traces
+for a ``tracegen`` service:
+
+    require 'ddtrace'
+
+    loop do
+	    Datadog.tracer.trace('hello-world') do |span|
+		    span.service = 'tracegen'
+		    span.resource = 'ruby'
+	    end
+	    sleep 1
```

Comment:
ah ok if that was the goal; the point is that usually people creates a span using a `sleep 1` to see something in the UI, and having the sleep inside our instrumentation block is the hello world everyone is going to use.